### ISSUE TITLE
Improvable loss printing for the WGAN critic

### ISSUE THREAD

**Issue by HitLuca** [eriklindernoren/Keras-GAN#32]
Two minor improvements to your WGAN code:
- The discriminator is called critic now, as it's not computing a probability anymore
- At line 149 you have
`d_loss = 0.5 * np.add(d_loss_fake, d_loss_real)`
and when printing the losses you have
` print ("%d [D loss: %f] [G loss: %f]" % (epoch, 1 - d_loss[0], 1 - g_loss[0]))`
this makes sense but you are essentially throwing away the information about all but the last critic training iteration. I suggest to average the complete critic training operation and print that instead, or add it along the other info.

**Comment by liuwei16** [comment]
But the quality of the generated MNIST is far from satisfactory, How to improve it?

**Comment by HitLuca** [comment]
You need to be quite a lot more specific for anyone to be able to help you. Why isn't the quality as good as expected? What is your experimental setup? How long was the training? Let me know

**Comment by eriklindernoren** [comment]
@HitLuca I have renamed the 'discriminator' to 'critic' (see 0293a4e) as this is the term they use in the paper. As for the loss I have decided to print the loss of the current iteration in all of the implementations so far. I might change this down the line as it is helpful to see what the loss is over the last 100 iterations or so. But I will keep this as is for now.
